core-feature-pack/src/main/resources/configuration/domain/template.xml

```diff
@@ -24,7 +24,7 @@
   ~ */
   -->
 
-<domain xmlns="urn:jboss:domain:4.1">
+<domain xmlns="urn:jboss:domain:4.2">
```

Comment:
I think submitting upstream first could make more sense, a lot of these changes will not have equivalents upstream.